nautobot/dcim/tests/test_models.py

```diff
@@ -2378,19 +2378,58 @@ def test_tagged_vlan_raise_error_if_mode_not_set_to_tagged(self):
         )
 
     def test_error_raised_when_adding_tagged_vlan_with_different_location_from_interface_parent_location(self):
+        intf_status = Status.objects.get_for_model(Interface).first()
+        intf_role = Role.objects.get_for_model(Interface).first()
+        location_type = LocationType.objects.get(name="Campus")
+        parent_location = Location.objects.filter(children__isnull=False, location_type=location_type).first()
+        child_location = Location.objects.filter(parent=parent_location, location_type=location_type).first()
```

Comment:
does `child_location` *need* to be a `Campus` `location_type`, or is any child of `parent_location` sufficient?